pyrdp/player/ReplayTab.py

```diff
@@ -88,3 +89,13 @@ def clear(self):
     def onClose(self):
         self.thread.close()
         self.thread.wait()
+
+    def mainWindowResized(self, width: int, height: int):
+        """
+        Called when the main PyRDP window is resized to allow to scale the current
+        RDP session being displayed.
+        :param width: The new width of the main window
+        :param height: The new height of the main window
+        """
+        self.widget.scale((height - self.text.height() - 200) / self.widget.height)
```

Comment:
What is this `200` for? I assume it's the height of the settings pane at the top? If that height is hardcoded, it should probably be a constant to avoid rescaling bugs if we ever change the GUI. In fact I would think that Qt has a way for you to query that information, so that the UI is responsive.

Reply:
With my current implementation it was hard to query this data, I'll see if it's easier with your suggestion

Reply:
not needed anymore